### ISSUE TITLE
On Mac the app window content displays as black

### ISSUE THREAD

**Issue by owenocq** [godotengine/godot#66653]
### Godot version

4.0 beta 1 and 4.0 beta 2 macos universal

### System information

OSX 10.14.6, 1.6 GHz Intel Core i5, Intel UHD Graphics 617 1536mb, Air 2018 retina display

### Issue description

All Godot windows do not display content (!). Windows can be repositioned, and resized but the content is black. This does not happen with Godot 3.5.

<img width="590" alt="black window - 'project manager'" src="https://user-images.githubusercontent.com/726551/193228639-c925c207-80d2-4d9e-9ec0-5200839aa483.png">

<img width="581" alt="black window - 'about godot'" src="https://user-images.githubusercontent.com/726551/193228633-fa342411-1c93-46dc-a421-331dc04e381e.png">

When opening Godot the OS X Console throws this error for the 'tccd' process: 

> "Prompting policy for hardened runtime; service: kTCCServiceAppleEvents requires entitlement com.apple.security.automation.apple-events but it is missing for ACC:{ID: org.godotengine.godot, PID[13559], auid: 501, euid: 501, binary path: '/Applications/Godot.app/Contents/MacOS/Godot'}, REQ:{ID: com.apple.appleeventsd, PID[64], auid: 55, euid: 55, binary path: '/System/Library/CoreServices/appleeventsd'}"

When quitting Godot, the OS X Console throws these 2 errors for the 'Godot' process: 

> "CMIOHardware.cpp:339:CMIOObjectGetPropertyData the System is exiting"

> "CMIO_DALA_System.cpp:264:GetPropertyData error 1970171760 (<private>) getting property selector (<private>) scope (<private>) element 0"`

The OS Console app has many other errors too but it's difficult to see which are triggered by Godot.

### Steps to reproduce

1. Open application. The windows shown all have black content for me.

### Minimal reproduction project

No project required.

**Comment by owenocq** [comment]
That fixes the issue for me. Thank you.

**Comment by oliver20033549** [comment]
i am using a high sierra version 10.13.6 and all i can see is grey

**Comment by Calinou** [comment]
@owenocq Can you check if this is resolved on your end on 4.0.beta5? If you get a gray screen instead, see the issue I linked below.

> i am using a high sierra version 10.13.6 and all i can see is grey

This is a different issue: https://github.com/godotengine/godot/issues/68760

**Comment by oliver20033549** [comment]
Yes I tried both --rendering-method mobile and forward_plus and it worked but when i tried to add a mesh or a camera there is no gizmo or mesh visible the material is blank to

**Comment by owenocq** [comment]
@Calinou Using 4.0.beta5 without the command line start:
• the initial project manager window now displays *correctly*
• but after creating a new project the project window is still black
• and the OSX Godot > About window is also still black

**Comment by clayjohn** [comment]
Can someone please test again on RC3? I think this was fixed as of Beta 9

**Comment by owenocq** [comment]
@clayjohn I've tested on RC1 and RC4, not working, same behaviour as Nov 2022

**Comment by GangstersPairOfDice** [comment]
## Godot Version

Godot Engine v4.03.stable.official [5222a99f5] installed through Steam on an external hard-drive 

## System Information:

MacBook Air (13-inch, Early 2015)
macOS 10.14.6
1.6 GHz Intel Core i5
8 GB 1600 Mhz DDR3
Intel HD Graphics 6000 1536 MB

## Issue Description

Was having the same issue as @owenocq where all Godot menus past the point of creating and opening a project would be completely black.

## Steps to reproduce

Open Godot. Create project. Open project.

## Solution

Was fixed using @Calinou 's suggestion of using the command line arguments:
--rendering-driver opengl3 --single-window

## Solution Reproduction

Go to Steam Library. Right Click on Godot Engine. Click on Properties. Click on General. Under Launch Options, paste in the command line arguments:
--rendering-driver opengl3 --single-window

## Bugs

If I have a Godot window and a Steam window open on two separate [workspaces](https://support.apple.com/guide/mac-help/work-in-multiple-spaces-mh14112/mac), everytime I go to the workspace where Godot is in, it will switch my workspace to the one where Steam is in.

**Comment by bruvzg** [comment]
> macOS 10.14.6

It's highly unlikely Vulkan renderer will work on anything older than 10.15.
